### PR TITLE
added kinds of services

### PR DESCRIPTION
**Pull request by ketrinkh**

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -299,4 +301,95 @@ h1, h2, h3, h4, h5, h6 {
...
+    height: 650px;
+
+    background: #ffffff;    
+    background-size: cover ;
+
+    padding-top: 80px;
+    top: 0%;
+    left: 0%;
+    right: 0%;
+    z-index: 1000;
```
Comment (viktordulo) [inline assets/css/style.css]: why do you need this property? (z-index)

File: assets/css/style.css [hunk 2]
```diff
@@ -299,4 +301,95 @@ h1, h2, h3, h4, h5, h6 {
...
+}
+
+.s2__inner {
```
Comment (viktordulo) [inline assets/css/style.css:365]: Not correct name of the class. It's better smth like s2__services-header

File: assets/css/style.css [hunk 3]
```diff
@@ -299,4 +301,95 @@ h1, h2, h3, h4, h5, h6 {
...
+    width: 60px;
+    height: 3px;
+
+    background-color: #e78080 ;
+    margin: 50px auto 50px;
+}
+
+/* S2_Services */
+
+.services1 {
```
Comment (viktordulo) [inline assets/css/style.css:401]: This is almost useless class in your code. You can use only border here. Flex isn't working here

File: index.html [hunk 4]
```diff
@@ -98,12 +98,69 @@ <h5 class="s1__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, se
...
+                    <div class="serv">
+                        <h3>Web design</h3>Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+                    </div>
+
+                    <div class="serv">
+                        <img src="assets/images/COMPUTER _ OK.png" alt="3">
+                    </div>
+                    <div class="serv">
+                        <h3>Creativity</h3>Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+                    </div>
```
Comment (viktordulo) [inline index.html]: You should combine img and text inside one div, not separated.

File: index.html [hunk 5]
```diff
@@ -98,12 +98,69 @@ <h5 class="s1__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, se
...
+                        <h3>Seo</h3>Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.
+                    </div>
+
+                    <div class="serv">
+                        <img src="assets/images/HOME.png" alt="5">                        
+                    </div>
+                    <div class="serv">
+                        <h3>Css/Html</h3>Lorem dolor sit amet, consectetur adipiscing elit, sed do tempor.
+                    </div>
+
```
Comment (viktordulo) [inline index.html]: You should combine img and text inside one div, not separated.

**Review by viktordulo — CHANGES_REQUESTED** [review]
Check comments and make fixes

File: index.html (hunk 4 above)
Comment (ketrinkh) [reply index.html]: I tried so. But then the text is placed below the picture, and should be placed next to the picture.

**Review by viktordulo — COMMENTED** [review]
Good job. Now try to resolve conflicts. Move your code to another line which isn't crossing with the code from master branch. Do it for index.html and style.css

**Review by viktordulo — APPROVED** [review]

**Pull request merged**